### PR TITLE
rules for OCP-25817

### PR DESCRIPTION
**Pull request by yapei**
@yanpzhan  Please help review. thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ create_resource_by_default_yaml:
       selector:
         xpath: //a[text()='Edit YAML']
     ref: click_edit_yaml_link
+  action: wait_until_no_loading
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Sometimes yaml editor loading need more time

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 2]
```diff
@@ -206,6 +211,11 @@ click_link_with_text:
     selector:
       xpath: //a[contains(.,'<text>') and contains(@href,'<link_url>')]
     op: click
+click_on_resource_item:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: click on any resource item with given name

**Comment by pruan-rht** [comment]
@yapei conflicts...pls rebase

**Comment by yapei** [comment]
rebased @yanpzhan  Can you help review?

**Comment by yapei** [comment]
@pruan-rht  Can you help merge if no objections?

**Pull request merged**